source/agora/common/crypto/Schnorr.d

```diff
@@ -53,15 +51,15 @@ import std.algorithm;
 import std.range;
 
 
-/// Single signature example
+// Single signature example
 nothrow @nogc @safe unittest
 {
     Pair kp = Pair.random();
     auto signature = sign(kp, "Hello world");
     assert(verify(kp.V, signature, "Hello world"));
 }
 
-/// Multi-signature example
+// Multi-signature example
```

Comment:
What I meant before is that if adding test-cases for bugs, those do not have to be documented. A `///` unittest is documented. A `\\` test is not.

However these existing tests do document the usage. They should stay documented.

Reply:
ok I see